patches/v8/0005-Speed-up-V8-bazel-build-by-always-using-target-cfg.patch

```diff
@@ -1,18 +1,11 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Felix Hanau <felix@cloudflare.com>
-Date: Wed, 7 Jun 2023 21:40:54 -0400
+From 1906956a221634250ed2ae32d2dae9d4ef2642a9 Mon Sep 17 00:00:00 2001
+From: Erik Corry <erikcorry@chromium.org>
+Date: Wed, 26 Feb 2025 18:33:33 +0100
 Subject: Speed up V8 bazel build by always using target cfg
 
-See the workerd build cfg changes for rationale. This provides a significant
```

Comment:
nit: Preserve PR descriptions (the second paragraph here is technically obsolete but the first still applies)